cla_public/apps/contact/fields.py

```diff
@@ -76,7 +80,9 @@ def process_formdata(self, valuelist):
 
     @classmethod
     def _format(cls, value):
-        return '{:%Y%m%d}'.format(value)
+        if value and not isinstance(value, basestring):
```

Comment:
I didn't see which class this method was in. But it's already assuming the value is a date with the hardcoded format string.